docs/DXIL.rst

```diff
@@ -267,6 +267,17 @@ The attribute 'waveops-include-helper-lanes' is utilized to indicate that wave o
 
 'fp32-denorm-mode' is employed to define the denorm mode for the function. The possible values for this attribute can be 'any', 'preserve', or 'ftz'.
 
+Module Flags
+------------
+
+DXIL uses `Module Flags Metadata <LangRef.html#module-flags-metadata>`_ to convey additional information about the module.
+
+The only module flags allowed in DXIL are "Dwarf Version" and "Debug Info Version".
+
+The "Dwarf Version" module flag is used to specify the DWARF version to use for debug information.
+
+The "Debug Info Version" module flag is used to specify the debug info schema version.
+
```

Comment:
For me, it'd help to see an example.  Are those names exactly how they'd appear in the metadata? Are these flags always i32's?

Something like:

```llvm
!0 = !{  i32 1, !"Dwarf Version", i32 1 }
```

Writing out that example made me realize that maybe we need to say something about the behaviors that someone consuming DXIL would expect to see here?

Reply:
Updated.